packages/component-submission/client/pages/DetailsStepPage.js

```diff
@@ -11,122 +12,138 @@ import SubjectAreaDropdown from '../components/SubjectAreaDropdown'
 import ExpandingTextField from '../components/ExpandingTextField'
 import OptionalSection from '../components/OptionalSection'
 
-const DetailsStep = ({ values, setFieldValue, setFieldTouched }) => (
-  // TODO: turn this ValidatedField into a richTextBox
-  <React.Fragment>
-    <Box mb={3}>
-      <ValidatedField
-        component={ExpandingTextField}
-        data-test-id="manuscript-title-editor"
-        label="Manuscript Title"
-        maxRows={4}
-        name="meta.title"
-        placeholder="Manuscript title"
-      />
-    </Box>
+const useSuggestions = ({ suggestions }) =>
+  suggestions.reduce(
+    (acc, sugg) => ({
+      ...acc,
+      [sugg.fieldName]: last(sortBy(sugg.suggestions, ['score'])).value,
```

Comment:
This should probably be sorted by time to select the most recent. Not by score

Reply:
`score` at the moment is basically just a sequential index of a suggestion